src/appfl/config/fed/federated.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
+from omegaconf import DictConfig, OmegaConf
+import torch
+""" various server updates are available 
```

Comment:
We need more description here. I don't understand how to use this configuration file. I feel this is a bit different from the other configuration files (e.g., `fedavg.py`). But, .. not sure...

Reply:
Do you feel that we should include a brief description on each algorithm?

Reply:
maybe a sentence for each algorithm?

Reply:
done